### ISSUE TITLE
Dark theme.

### ISSUE THREAD

**Issue by BossXD** [BullyWiiPlaza/JGeckoU#67]
How bout a dark theme?
Not really needed, but it would be easier on the eyes at night.. obviously.

**Comment by BullyWiiPlaza** [comment]
Yeah, that's possible but it's not so important so I prefer working on other things first. I'll keep it on my TODO list though.
